js/ui/form/ui.form.js

```diff
@@ -1599,7 +1616,17 @@ var Form = Widget.inherit({
      * @return any
      */
     getEditor: function(dataField) {
-        return this._editorInstancesByField[dataField];
+        return this._instanceStorage.findByDataField(dataField) || this._instanceStorage.findByName(dataField);
+    },
+
+    /**
+     * @name dxFormMethods.getButton
+     * @publicName getButton(name)
+     * @param1 name:string
+     * @return any
+     */
+    getButton: function(name) {
```

Comment:
@TikhonovaTatyana What do you think, maybe better we'll add neutral `getWidget`, and marked `getEditor` as deprecated in 18.2?

Reply:
I like smth like getItemWidget more than 2 separate methods

Reply:
I've talked with Roma, he likes the variant with getButton